code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -375,3 +380,15 @@
 	desc = "A medicine used to treat genetic conditions, including benign and malignant tumours."
 	icon_state = "pill11"
 	reagents_to_add = list(/singleton/reagent/ryetalyn = 10)
+
+/obj/item/reagent_containers/pill/caffeine
+	name = "5u Caffeine Pill"
+	desc = "A stimulant used to help with alertness and fatigue."
+	icon_state = "pill18"
+	reagents_to_add = list(/singleton/reagent/caffeine = 5)
+
+/obj/item/reagent_containers/pill/nicotine
+	name = "5u Nicotine Pill"
+	desc = "A pull used to treat nicotine addiction."
```

Comment:
minor spelling mistake 🔥🔥🔥🔥🔥🔥🔥

Reply:
DONE still wish it was patches but porting touch-based chems would be way too much hastle